packages/block-library/src/style.scss

```diff
@@ -133,7 +133,36 @@
 	font-size: 36px;
 }
 
-.has-larger-font-size, // not used now, kept because of backward compatibility.
+.has-larger-font-size, // Not used now, kept because of backward compatibility.
 .has-huge-font-size {
 	font-size: 42px;
 }
+
+
+/**
+ * Vanilla Block Editor Styles
```

Comment:
This here is inspired by feedback on a separate but related PR, here: https://github.com/WordPress/gutenberg/pull/14366#issuecomment-471604552

The idea is that we know we want to apply a modicum of styles to blocks, either to support them structurally (function in the first place), or to make sure we don't provide a broken or unstyled-looking experience. In the case of the `figcaption` element as is the core of that other PR, this element is born without margins, which means it sits right up next to the image unless it receives a margin.

The question is, should Gutenberg have such a base stylesheet that cascades? I've collected the two existing rules we already have, for `img` and `iframe`, and once that other PR goes through, we'd also have `figcaption` on this list. Should `p` also be there? And should there be a way to opt-in to this vanilla stylesheet, or opt _out_ of it? These are questions that would be good to figure out, and I would appreciate your thoughts.

Reply:
> should Gutenberg have such a base stylesheet that cascades?

I'm not clear on the implications of this either way. Or is that the question?

Are you proposing effectively a `normalize.css` style base stylesheet for the Editor that just restores the default styles in a standard manner or would this go further still?

> And should there be a way to opt-in to this vanilla stylesheet, or opt out of it?

You should certainly be able to disable it if you know what you're doing. Probably on by default.

Reply:
I think the biggest group of themes is the ones that created a style sheet for the old editor. The best way to move forward with the least impact is to have the `add_editor_style` function call the new block editor style function that loads the styles with the prefixed selectors. Then the old styles will be in effect for the editor and the front end, and **no blocks are styled**. The HTML elements themselves are styled. This is the majority of existing themes. The editor should only style the UI elements and not the blocks. If any blocks need styling, put it in the style sheet that is opt-in.
With the old editor, there were rudimentary styles for standard classes `alignleft`. `alignright`, `aligncenter`. These were not output on the front end. The theme is responsible for that, and it makes sense to continue in the same way.